rivershared/levenshtein/License.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 Agniva De Sarker
```

Comment:
Trying to confirm if this is the correct copyright holder for this bit, where did this name come from?

Reply:
I vendored the code originally from this repo:

https://github.com/agnivade/levenshtein

I'm thinking I may write my own implementation from scratch at some point when I have a bit more time. I ended up rebuilding a lot of the code anyway to modernize and satisfy lint.

Reply:
I added a comment pointing to the source repo.